lib/zendesk_apps_support/app_version.rb

```diff
@@ -31,7 +38,11 @@ def valid_for_update?
     end
 
     def deprecated?
-      @version == DEPRECATED
+      DEPRECATED.include?(@version)
```

Comment:
Considering this is a public constant I think it would be dangerous to change its type. See https://github.com/zendesk/zendesk_app_market/search?utf8=%E2%9C%93&q=AppVersion&type=.